Formula/thrax.rb

```diff
@@ -28,19 +29,15 @@ class Thrax < Formula
   depends_on "libtool" => :build
 
   depends_on "openfst"
-
-  on_linux do
-    depends_on "gcc"
-    depends_on "python@3.10"
-  end
+  depends_on "python@3.10"
```

Comment:
Why not
```ruby
uses_from_macos "python"
```
?

Reply:
I'll change to that.  There's no reason I am aware of that it has to be a specific version of Python 3.